src/app.ts

```diff
@@ -71,6 +76,64 @@ const main = async () => {
     res.json({ success: 'true' });
   });
 
+  const storage = multer.memoryStorage();
+  const upload = multer({ storage: storage });
+
+  // Not really sure where in the code base go, but here's the working route!
+  // I did my part :)
+  // Is just using `/flyers/` the right route here?
```

Comment:
Yeah `/flyers/` is fine, I think this location is fine for now since I don't foresee us adding more flyer functionality. If we do, we'll be able to justify having a separate folder for flyer code and can refactor this then